content/applications/marketing/events/create_events.rst

```diff
@@ -0,0 +1,427 @@
+   To add an optional :guilabel:`Description` column to the :guilabel:`Tickets` tab, click the
+   :guilabel:`additional options` :icon:`oi-settings-adjust` menu, located to the far-right of the
+   column titles.
+
+   Then, tick the checkbox beside :guilabel:`Description` from the resulting drop-down menu.
+
+   When added, the option to add brief descriptions for each event ticket appears, which can be used
+   to inform attendees of any perks or amenities that may coincide with specific ticket purchases.
+
+.. _create_events/event_form/communication:
+
+Communication tab
+-----------------
+
+In the :guilabel:`Communication` tab of an event form, create various marketing communications that
+can be scheduled to be sent at specific intervals leading up to, and following, the event.
+
+.. image:: create_events/communication-tab.png
+   :align: center
+   :alt: Typical communication tab on an event form in the Odoo Events application.
+
+.. note::
+   By default, Odoo provides three separate pre-configured communications on every new event form.
+   One is an email sent after each registration to confirm the purchase with the attendee. The other
+   two are email event reminders that are scheduled to be sent at different time intervals leading
+   up to the event to remind the recipient of the upcoming event.
+
+To add a communication in the :guilabel:`Communication` tab, click :guilabel:`Add a line`. Then,
+select the desired type of communication in the :guilabel:`Send` field. The options are:
+:guilabel:`Mail`, :guilabel:`SMS`, :guilabel:`Social Post`, or :guilabel:`WhatsApp`.
+
+.. important::
+   The :guilabel:`Social Post` option **only** appears if the *Social Marketing* application is
+   installed. The :guilabel:`WhatsApp` option **only** appears if the *WhatsApp Integration* module
+   is installed.
+
+   :doc:`WhatsApp <../../productivity/whatsapp>` templates **cannot** be edited during active
+   configuration. A separate approval from *Meta* is requried.
+
+Then, either select a communication template from the :guilabel:`Template` drop-down menu.
+
+.. note::
+   These existing communication templates can be modified directly from this field, if necessary, by
+   selecting the chosen template, and clicking the :guilabel:`➡️ (right arrow)` icon to the right.
+   Doing so reveals a separate page where users can edit the :guilabel:`Content`, :guilabel:`Email
+   Configuration`, and :guilabel:`Settings` of that particular communication template.
+
+.. tip::
+   To create a new communication template from the :guilabel:`Template` field in the
+   :guilabel:`Communication` tab, start typing in the name of the new template, then click
+   :guilabel:`Create and edit...` from the resulting drop-down menu. Doing so reveals a blank
+   :guilabel:`Create Template` pop-up window, wherein users can create a custom communication
+   template.
+
+   To learn more, refer to the :ref:`Create an email <email_marketing/create_email>` documentation.
```

Comment:
The email templates in the Communications tab are not marketing mass mailings (from the Email Marketing application), they are transactional email templates defined in the Email section of the Settings app. 

I add more info in the following suggestions :)
```suggestion
```